sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -664,8 +664,8 @@ AZ_NODISCARD az_result az_iot_pnp_client_property_builder_end_reported_status(
  * az_iot_pnp_client_property_get_next_component_property().
```

Comment:
Can you please update the warning docs based on https://github.com/Azure/azure-sdk-for-c/pull/1425#discussion_r505718067 as well.